src/components/PermissionsManagement/PermissionListItem.jsx

```diff
@@ -167,7 +171,8 @@ function PermissionListItem(props) {
               }}
               disabled={
                 // eslint-disable-next-line react/destructuring-assignment
-                !props.hasPermission('putRole') || immutablePermissions.includes(permission)
+                !(props.hasPermission('putRole') || props.hasPermission('postRole')) ||
+                immutablePermissions.includes(permission)
```

Comment:
```suggestion
                immutablePermissions.includes(permission)
```
Wouldn't work for individual permissions.